.github/workflows/release-helm-chart.yaml

```diff
@@ -1,9 +1,10 @@
-name: Release Charts
```

Comment:
nit: should we merge both commits related to the release of helm chart

Reply:
Should these workflows depends on `Helmfile lint` job, to avoid releasing a broken chart?